examples/dreamcast/network/httpd/httpd.c

```diff
@@ -299,7 +311,8 @@ void *client_thread(void *p) {
     fs_close(f);
 
 out:
-    free(buf);
+    if(buf != NULL)
```

Comment:
Was this flagged or something? Because it's actually valid to `free()` a `NULL` in C/C++. It's just a nop, essentially.

Reply:
Whoops! Nope, what was flagged with something else in the same path and I just had the possibility of double-free rather than freeing NULL in mind -_-; Good catch.